examples/flutter_gallery/lib/demo/colors_demo.dart

```diff
@@ -7,55 +7,112 @@ import 'package:flutter/material.dart';
 const double kColorItemHeight = 48.0;
 
 class Palette {
-  Palette({ this.name, this.primary, this.accent, this.threshold = 900});
+  Palette({this.name, this.primary, this.accent, this.threshold = 900});
 
   final String name;
   final MaterialColor primary;
   final MaterialAccentColor accent;
-  final int threshold; // titles for indices > threshold are white, otherwise black
+  final int
+      threshold; // titles for indices > threshold are white, otherwise black
 
   bool get isValid => name != null && primary != null && threshold != null;
 }
 
 final List<Palette> allPalettes = <Palette>[
-  Palette(name: 'RED', primary: Colors.red, accent: Colors.redAccent, threshold: 300),
-  Palette(name: 'PINK', primary: Colors.pink, accent: Colors.pinkAccent, threshold: 200),
-  Palette(name: 'PURPLE', primary: Colors.purple, accent: Colors.purpleAccent, threshold: 200),
-  Palette(name: 'DEEP PURPLE', primary: Colors.deepPurple, accent: Colors.deepPurpleAccent, threshold: 200),
-  Palette(name: 'INDIGO', primary: Colors.indigo, accent: Colors.indigoAccent, threshold: 200),
-  Palette(name: 'BLUE', primary: Colors.blue, accent: Colors.blueAccent, threshold: 400),
-  Palette(name: 'LIGHT BLUE', primary: Colors.lightBlue, accent: Colors.lightBlueAccent, threshold: 500),
-  Palette(name: 'CYAN', primary: Colors.cyan, accent: Colors.cyanAccent, threshold: 600),
-  Palette(name: 'TEAL', primary: Colors.teal, accent: Colors.tealAccent, threshold: 400),
-  Palette(name: 'GREEN', primary: Colors.green, accent: Colors.greenAccent, threshold: 500),
-  Palette(name: 'LIGHT GREEN', primary: Colors.lightGreen, accent: Colors.lightGreenAccent, threshold: 600),
-  Palette(name: 'LIME', primary: Colors.lime, accent: Colors.limeAccent, threshold: 800),
+  Palette(
```

Comment:
this is a case where we should probably manually opt in to trailing comma syntax even though the original source code didn't suggest it.